cmd/zerosum/main.go

```diff
@@ -149,6 +149,15 @@ func (z *zeroSum) accountsLen() int {
 	return len(z.accounts.m)
 }
 
+func (z *zeroSum) maybeLogError(err error) {
+	if strings.Contains(err.Error(), "range is frozen") {
+		time.Sleep(time.Second)
```

Comment:
Do you mean, why am I sleeping here? If I don't, we loop immediately trying to send another request. I suppose that's ok, though. What I really wanted to avoid was the log message which was very spammy. I'll take this out.